src/test/java/com/snowflake/kafka/connector/internal/TestUtils.java

```diff
@@ -731,4 +735,14 @@ public static void checkTableContentOneRow(String tableName, Map<String, Object>
   public static String getExpectedLogTagWithoutCreationCount(String taskId, int taskOpenCount) {
     return Utils.formatString(TASK_INSTANCE_TAG_FORMAT, taskId, taskOpenCount, "").split("#")[0];
   }
+
+
+  public static SnowflakeStreamingIngestClient createStreamingClient(Map<String, String> config, String connectorName) {
```

Comment:
Have to explicitly create a client because a majority of the integration tests rely on an active client. If we are actively creating a client, I'd rather label them end to end tests